app/src/main/java/de/danoeh/antennapod/fragment/ExternalPlayerFragment.java

```diff
@@ -78,7 +79,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onActivityCreated(Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
         butPlay.setOnClickListener(v -> {
-            if (controller != null) {
+            if (controller != null && controller.getMedia().getMediaType() == MediaType.VIDEO) {
+                controller.playPause();
+                if(controller.getStatus() != PlayerStatus.PLAYING) {
+                    getContext().startActivity(PlaybackService.getPlayerActivityIntent(getContext(),controller.getMedia()));
+                }
+            }else if (controller != null) {
```

Comment:
Something like the following ensures that there is no race condition where calling playPause and getting the player status interfere.

```
if (controller != null) {
    if (video and not playing) {
        playPause() // Do not call before checking player status but not after starting activity
        start activity
    } else {
        playPause();
    }
}
```